examples/js/archive.js

```diff
@@ -0,0 +1,131 @@
+let map;
```

Comment:
Hi all, this looks good, but @7malikk would you be able to check this file against the most recent changes in https://github.com/publiclab/Leaflet.DistortableImage/pull/1286/files, or have you already? I just want to be sure we don't lose that last change regarding the search box. Thanks!

Reply:
Just went through it again, seems @Aarav238 might've missed the changes regarding the search box.

@Aarav238 would you be willing to fetch the most recent changes files? 

Thank you @jywarren for that pointer.